dnase_partial.wdl

```diff
@@ -0,0 +1,42 @@
+version 1.0
+
+
+import "wdl/workflows/dnase_replicate.wdl" as process
+import "wdl/workflows_partial/dnase_replicate_no_footprints.wdl" as process_no_footprints
+
+
+workflow dnase {
+    input {
+        Array[Replicate] replicates
+        Boolean omit_footprints = false
```

Comment:
Why not just make this `dnase_no_footprints.wdl` and call `dnase_replicate_no_footprints`? Can avoid the boolean logic, will just know you are skipping footprints by calling this WDL. Also don't think we should allow this second pathway of calling normal WDL.

Reply:
Yeah, now that the `preseq_defects_mode` is embedded in `replicate` we can do this.